spec/models/ingredient_spec.rb

```diff
@@ -12,4 +12,25 @@
       it { should have_many(:recipes).through(:recipe_ingredients) }
    end
 
+    describe "total_cost" do
+      it "can total the cost of ingredients" do
+        ingredient_1 = Ingredient.create!(name: "Garlic Powder", cost: 1)
+        ingredient_2 = Ingredient.create!(name: "Beans", cost: 3)
+        
+        expect(Ingredient.total_cost).to eq(4)
```

Comment:
Since this is a class method and you're using it on an instance's association in your feature test, that you also model that behavior in this model test. I think it would be easier to implement as a Recipe's instance method instead.